src/main/integrations/companion-server/api-shared/errors.ts

```diff
@@ -21,6 +21,7 @@ export const InvalidVolumeError = createError<[number]>("INVALID_VOLUME", "Volum
 export const InvalidRepeatModeError = createError<[RepeatMode]>("INVALID_REPEAT_MODE", "Repeat mode '%s' cannot be set", 400);
 export const InvalidPositionError = createError<[number]>("INVALID_SEEK_POSITION", "Seek position '%s' is invalid", 400);
 export const InvalidQueueIndexError = createError<[number]>("INVALID_QUEUE_INDEX", "'%s' is an invalid queue index for the current queue", 400);
+export const InvalidSongRequestError = createError<[]>("INVALID_SONG_REQUEST", "Song and Playlist cannot both be empty", 400);
```

Comment:
`InvalidChangeVideoRequestError` to align with requested change.

Recommended error details should be `"'id', 'playlistId', or both must be provided"`